graphql-apollo-server-tutorial.md

```diff
@@ -3048,7 +3039,7 @@ const createUsersWithMessages = async date => {
 };
 {{< /highlight >}}
 
-That's it for the cursor which will be the creation date of each message. Now, let's advance the previous pagination to cursor-based pagination in the *src/schema/message.js* file. You only need to exchange the offset with the cursor. So instead of having an offset which can only be matched implicitly to an item in a list (and changes once an item is deleted from the list), the cursor has a stable position within the list, because the creation dates of the messages will not change.
+That's the cursor the will be the creation date of each message. Now we have to change the original pagination to cursor-based in the *src/schema/message.js* file. You only need to exchange the offset with the cursor. Instead of an offset that can only be matched implicitly to an item in a list and changes once an item is deleted from the list, the cursor has a stable position within, because the message creation dates won't change.
```

Comment:
"That's the cursor the will be the creation date of each message." <- I think there is something wrong with this sentence.